torch/_functorch/partitioners.py

```diff
@@ -511,7 +510,9 @@ def object_nbytes(x) -> int:
             return 0
         return _tensor_nbytes(hint_int(x.numel(), fallback=4096), x.dtype)
 
-    if "val" in node.meta:
+    if node.op == "get_attr" or node.target is torch.ops.aten._assert_scalar.default:
```

Comment:
for torch.ops.aten._assert_scalar.default node.meta["val"] could be none so we check this first.